lib/Parser/cmperr.cpp

```diff
@@ -4,8 +4,16 @@
 //-------------------------------------------------------------------------------------------------------
 #include "ParserPch.h"
 
-ParseExceptionObject::ParseExceptionObject(HRESULT hr) : m_hr(hr)
+ParseExceptionObject::ParseExceptionObject(HRESULT hr, LPCWSTR stringOneIn, LPCWSTR stringTwoIn)
 {
+    m_hr = hr;
```

Comment:
why move this assignment from the constructor-initialization syntax to the body of the constructor?

Reply:
That wasn't a new change. There was no real reason for it though, I thought it looked neater next to the two string properties, want me to revert it?

Reply:
I don't **think** it makes any difference. I guess we don't have a code guideline or real consistency around this point AFAICT. @Penguinwizzard do you know if this matters in terms of code gen?

You can leave it alone for now, so we can let the CI finish and merge this if it doesn't make a difference.

Reply:
It's mostly stylistic, and there's no real enforced style in ChakraCore (yet, anyway). I'd be decently surprised if we get different codegen due to this in optimized builds.